scripts/install-jdk.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+. ~/.jabba/jabba.sh
+
+# Defaults to Java 1.8
+jdk_to_install=${TRAVIS_JDK:-8}
+
+# Ensure the build will use the latest JDK for that major release
+jabba_identifier=$(jabba ls-remote adopt@ | grep "1.${jdk_to_install}" | sort -r | head -1)
```

Comment:
Simplified into one command which also allows us to capture the exit code.

```suggestion
jabba_identifier=$(jabba ls-remote "adopt@~1.${jdk_to_install}.0-0" --latest minor) || travis_terminate 1
```

Test run of what that one command outputs:

```bash
08:29:55 $ for jdk_to_install in 8 11; do
> jabba ls-remote "adopt@~1.${jdk_to_install}.0-0" --latest minor
> done
adopt@1.8.212-04
adopt@1.11.28-0
```

previous command:

```bash
08:30:23 $ for jdk_to_install in 8 11; do
> jabba ls-remote adopt@ | grep "1.${jdk_to_install}" | sort -r | head -1
> done
adopt@1.8.212-04
adopt@1.11.28-0
```